src/elitefurretai/utils/team_repo.py

```diff
@@ -13,14 +14,64 @@ class TeamRepo:
 
     def __init__(self, filepath: str = "data/teams"):
         self._teams: Dict[str, str] = {}
-        files = [
-            os.path.join(filepath, f)
-            for f in os.listdir(filepath)
-            if os.path.isfile(os.path.join(filepath, f))
-        ]
-        for file in files:
-            with open(file, "r") as f:
-                self._teams[os.path.basename(file).replace(".txt", "")] = f.read()
+
+        for format_folder in os.listdir(filepath):
+            format_name = format_folder
+            folder_path = os.path.join(filepath, format_folder)
+            if os.path.isdir(folder_path):
+                print(f"format name: {format_name}")
+                filenames = os.listdir(folder_path)
+                for filename in filenames:
+                    team_path = os.path.join(filepath, format_folder, filename)
+                    print(team_path)
+                    with open(team_path, "r") as f:
+                        team_string = f.read()
+                    self._teams[
```

Comment:
Thinking more about it, I would lightly prefer it if we stored this in a Dict of Dicts? This way, our data structure mirrors our file structure?

So, to get all teams for a format, we could easily do: `teamrepo.teams[format].vaues()`, and to get a specific team, we could do: `teamrepo.teams[format][team_name]`

Reply:
This is a great idea, will do.